bigquery/api/sync_query.py

```diff
@@ -66,7 +66,7 @@ def main(project_id, query, timeout, num_retries):
             pageToken=page_token,
             **query_job['jobReference']).execute(num_retries=2)
 
-        print(json.dumps(page['rows']))
+        print(json.dumps(page.get('rows', [])))
```

Comment:
```
$ python3 -c 'import json ; print(json.dumps(None))'
null
```
I went with [] because the return value is an array containing all the result rows, which feels like it should be an empty array when there are no rows. But I am almost apathetic towards this :P as long as it doesn't KeyError, I'm fine.